app/gui/tsconfig.node.json

```diff
@@ -10,6 +10,7 @@
     "tailwind.config.ts",
     "integration-test/**/*",
     "src/dashboard/modules/payments/constants.ts",
+    "src/dashboard/providers/FeatureFlagsProvider.tsx",
```

Comment:
Just wondering, why is it needed?

Reply:
no clue :skull: but without it it seems to cause errors - which are, admittedly, not an issue in CI because the lints aren't checked, but i imagine one of the modules here *indirectly* imports this file?

Reply:
likely `api.ts` possibly